vtysh/vtysh.c

```diff
@@ -1132,6 +1142,26 @@ DEFUNSH(VTYSH_BGPD, address_family_ipv4, address_family_ipv4_cmd,
 	return CMD_SUCCESS;
 }
 
+DEFUNSH(VTYSH_BGPD, address_family_flowspecv4, address_family_flowspecv4_cmd,
+	"address-family ipv4 flowspec",
+	"Enter Address Family command mode\n"
+	"Address Family\n"
+	"Address Family Modifier\n")
+{
+	vty->node = BGP_FLOWSPECV4_NODE;
+	return CMD_SUCCESS;
+}
+
+DEFUNSH(VTYSH_BGPD, address_family_flowspecv6, address_family_flowspecv6_cmd,
+	"address-family ipv6 flowspec",
+	"Enter Address Family command mode\n"
+	"Address Family\n"
+	"Address Family Modifier\n")
+{
+	vty->node = BGP_FLOWSPECV6_NODE;
+	return CMD_SUCCESS;
+}
+
```

Comment:
it would be nicer if this was a single DEFUNSH that picked up the afi from the command line.  (This is a non-blocking comment).

Reply:
This is not possible as of now because of a limitation in vtysh: Please see #358.